acceptance/config/windows-aio-2012-x86_64.cfg

```diff
@@ -0,0 +1,32 @@
+HOSTS:
+  ubuntu1204:
+    roles:
+      - master
+      - database
+      - dashboard
+    platform: ubuntu-12.04-amd64
+    template: ubuntu-1204-x86_64
+    hypervisor: vcloud
+  w2012-puppetx64:
+    ruby_arch: x64
+    roles:
+      - agent
+      - default
+    platform: windows-2012-x86_64
+    template: win-2012-x86_64
+    hypervisor: vcloud
+  w2012-puppetx86:
+    ruby_arch: x86
```

Comment:
See https://gist.github.com/Iristyle/dd3a992ffbf0f77430ad - `ruby_arch` is ignored when acquiring the MSI, and `puppet-agent-x64.msi` is installed every time.